Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.7.1
+// swift-tools-version:5.7.2
```

Comment:
While the release notes say it requires 5.7.2, their tools is still 5.7, so I think we can keep things as they are and let the compiler fail where it may fail.
```suggestion
// swift-tools-version:5.7.1
```